docs/UserGuide.md

```diff
@@ -174,22 +174,23 @@ Ward: A1
 Tags:
 ```
 
-### Locating patients by name: `find`
+### Locating patients either by name or by IC: `find`
 
 Finds patients whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find n\KEYWORD [MORE_KEYWORDS]`, `find ic\KEYWORD`
```

Comment:
Correct format should me `find [n\NAME]... [ic\IC]`

... indicates that this can accept 0 or more arguments.
I am not sure if ic allows multiple arguments as well.

Can refer to this:

![image](https://github.com/AY2324S2-CS2103T-F10-1/tp/assets/52852142/fea55122-884d-4c50-81b4-0d03a73f292b)